pkg/test/case.go

```diff
@@ -165,7 +200,26 @@ func (t *Case) Run(test *testing.T, tc *report.Testcase) {
 		}
 	}
 
-	t.CollectEvents(ns)
+	t.CollectEvents(ns.Name)
+}
+
+func (t *Case) determineNamespace() (*namespace, error) {
+	ns := &namespace{
+		Name:        t.Namespace,
+		AutoCreated: false,
+	}
+	//ns := t.Namespace
```

Comment:
Is this line a genuine comment or a remnant of debugging? If the former than I have to admit it's not very clear.